package.json

```diff
@@ -1,83 +1,20 @@
 {
-    "name": "{{APP_NAME}}",
+    "name": "{{APP_NAME}}-monorepo",
```

Comment:
not sure if i love the `-monorepo` naming here for the root package name 🤔 

I actually kinda like the root just being `{{APP_NAME}}`, and then the "app" workspace's package name would literally be `app`, to be consistent with "deployment" (and  "docs" eventually). This is basically how EF's monorepo is structured.

That has the added benefit of making local/relative imports across different apps more consistent too, in cases where we want to copy/paste code cross-apps (tho i think we are using the `~/../..` path syntax mostly, so its currently not a _huge_ deal in that respect.)